src/pages/categories/[category].tsx

```diff
@@ -48,7 +49,7 @@ const Category: NextPage = () => {
           my={12}
           gap={8}
         >
-          {[...Array(6)].map(i => (
+          {Array.from(new Array(6), (_x, i) => i).map(i => (
```

Comment:
You can do:
```suggestion
          {Array.from({ length: 6 }).map((_, i) => (
```
More readable and involves less declarations

Reply:
Awesome !